index.js

```diff
@@ -181,7 +181,7 @@ class EmailNotifier extends NotificationBase {
 
     // Validate the settings email object
     static validateConfig(config) {
-        return Joi.validate(config, SCHEMA_EMAIL);
+        return SCHEMA_EMAIL.validate(config);
```

Comment:
`SCHEMA_EMAIL` cannot be validate correctly.
In the case of the above, you have not verified the following
```
test:
  settings:
    email: ←Not verified.
```
It is better to use `SCHEMA_BUILD_SETTINGS`.

Reply:
If we want to call validate only if it have `settings.email`, I guess we can't use `SCHEMA_BUILD_SETTINGS`.
https://github.com/screwdriver-cd/config-parser/pull/105/files

Reply:
The role of the validator in this plugin is to return whether the email setting is valid.
If the following yaml may not work properly.

```
  email:
    requires: [ ~commit, ~pr ]
    image: cd.docker-registry.corp.yahoo.co.jp:4443/node/node12:latest
    settings:
      emaill ←typo:
        addresses:
          - imiyagam@yahoo-corp.jp
        statuses:
          - SUCCESS
          - FAILURE
```
When actually used, it works by simply passing the job `settings`.

Reply:
If `emaill` is made available by Custom Notifications, it is not a typo.
https://docs.screwdriver.cd/cluster-management/configure-api#notifications-plugin

I don't think there's a good way to handle it at the moment, we'll just have to verify the contents of the `settings.email`.

Reply:
Ahh, I was confusing `SCHEMA_BUILD_SETTINGS` and `SCHEMA_BUILD_DATA`. Probably it should use `SCHEMA_BUILD_SETTINGS`.